roles/cloud_connector/tasks/cleanup_receptor.yml

```diff
@@ -0,0 +1,37 @@
+---
+- name: Check if foreman-maintain installed
+  ansible.builtin.package_facts:
+    manager: auto
+
+- name: Unlock packages
+  command: "foreman-maintain packages unlock"
+  when: "'foreman-maintain' in ansible_facts.packages"
+
+- name: Stop any receptor services
+  command: systemctl stop receptor*
+
+- name: Disable any receptor services
+  command: systemctl disable receptor*
+  ignore_errors: true
```

Comment:
Should we find the services using [service_facts](https://docs.ansible.com/ansible/latest/collections/ansible/builtin/service_facts_module.html) instead and use the `service` module to stop/disable them?

Reply:
Plus, below, you purge `/etc/systemd/system/receptor@.service` but this here would also have stopped/disabled the `receptoral-unified-backup` service (sorry, if your keyboard now contains more coffee than before) ;-)